nfsstat/datadog_checks/nfsstat/nfsstat.py

```diff
@@ -38,9 +38,11 @@ def check(self, instance):
         all_devices = []
         this_device = []
         custom_tags = instance.get("tags", [])
-
         for l in stat_out.splitlines():
-            if not l:
+            if l.find(b'No NFS mount point') >= 0:
+                self.warning("No NFS mount points were found")
+                return
```

Comment:
why does this return while the next one just continues? seems like a contradiction of the logic?

Reply:
I didn't mean to include the change to line 45, that was part of what I was originally trying. I updated it.

Reply:
Now we properly continue on empty lines but if we see there aren't any mount points, we just end the check run.

Reply:
Okay!